core/trace.js

```diff
@@ -6,12 +6,26 @@ var Fiber = require('fibers');
 
 module.exports = function(message) {
```

Comment:
This is a little complex, but it allows us to indicate which process is being modified and which process (or the runtime) is doing the modification.